src/controllers/categoryController.ts

```diff
@@ -0,0 +1,74 @@
+import Category from '../models/Category';
+import {Request, Response, NextFunction} from 'express';
+
+async function retrieveCategory(req: Request, res: Response, next: NextFunction) {
+  try {
+    const category = await Category.findByPk(req.params.id, {
+      include: [{association: Category.associations.CategoryStalls}],
+    });
+    if (category === null) {
+      res.status(404).end();
+      return;
+    }
+    req.category = category;
+    next();
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function indexCategory(req: Request, res: Response, next: NextFunction) {
+  try {
+    const categories = await Category.findAll({
+      include: [
+        {
+          association: Category.associations.CategoryStalls,
+        },
+      ],
+    });
+    res.status(200).json(categories);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function showCategory(req: Request, res: Response, next: NextFunction) {
+  try {
+    res.status(200).json(req.category);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function createCategory(req: Request, res: Response, next: NextFunction) {
+  try {
+    const category = await Category.create(req.body);
```

Comment:
This should return with the `include` option for consistency across the API calls.

Reply:
For the other controllers there's no include option except for the Stall Controller